exercises/javascript-carousel/main.js

```diff
@@ -0,0 +1,68 @@
+var $left = document.querySelector('#left');
+var $right = document.querySelector('#right');
+var $images = document.querySelectorAll('.carouselImg');
+var $circleButtons = document.querySelector('.circle');
+var $circles = document.querySelectorAll('.fa-circle');
+
+var counter = 1;
+
+$right.addEventListener('click', rightArrowClicked);
+$left.addEventListener('click', leftArrowClicked);
+$circleButtons.addEventListener('click', circleClicked);
+
+function viewSwap() {
+  stopCarousel();
+  carouselID = setInterval(moveCarousel, 3000);
+  for (var i = 0; i < 5; i++) {
+    if (counter === parseInt($images[i].getAttribute('data-view'))) {
+      $images[i].className = 'carouselImg';
+      $circles[i].className = 'fas fa-circle';
+    } else {
+      $images[i].className = 'hidden';
+      $circles[i].className = 'far fa-circle';
+    }
+  }
+}
+
+function circleClicked(event) {
+  if (event.target.className.includes('far')) {
+    counter = (parseInt(event.target.getAttribute('data-view')));
+    viewSwap();
+  }
+}
+
+function rightArrowClicked(event) {
+  if (counter === 5) {
+    counter = 1;
+    viewSwap();
+    return;
+  }
+  counter += 1;
+  viewSwap();
+}
+
+function leftArrowClicked(event) {
+  if (counter === 1) {
+    counter = 5;
+    viewSwap();
+    return;
+  }
+  counter -= 1;
+  viewSwap();
+}
+
+function moveCarousel() {
+  if (counter === 5) {
+    counter = 1;
+    viewSwap();
+    return;
+  }
+  counter += 1;
+  viewSwap();
+}
```

Comment:
You already have a function with this exact work named `rightArrowClicked`. You might rename that function to `moveCarouselForward` and rename `leftArrowClicked` to `moveCarouselBackward`.